react-class-components/src/index.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-useless-constructor */
```

Comment:
Instead of turning off the ESLint rule, you can just delete your class's `constructor` method.